recipes/pysurvival/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pysurvival" %}
+{% set version = "0.1.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 57cb567323fa314ceaf17fe39745b007855606cb1304a4124a78334ff9d03707
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py>38 or win]
```

Comment:
While you are certainly free to do what you want with your feedstock, it is rather strange that you are skipping 3.9 and 3.10.

It may be that you can delete some of the generated cython files and regenerate them here:
https://github.com/square/pysurvival/issues/43

There has been some back and forth between including the generated cython files in the "pip" source and not.

Something like what I did here:
https://github.com/conda-forge/fa2-feedstock/blob/master/recipe/meta.yaml#L16

Might get you unstuck for newer versions of python

For your case, you might want
```
rm -f pysurvival/cpp_extensions/_*.cpp
```

Though if you prefer, you can remove them manually:
![image](https://user-images.githubusercontent.com/90008/153781108-9e154fec-5de0-43e0-9119-0e62cf9fde80.png)

Reply:
One of the advantages of included the generated c files is that you are more sure of what you are asking the user to compile.

One disadvantage is that you are not always forward compatible with newer versions of cpython (as is the case here).

Reply:
So this does not work...

Reply:
Adding this resulted in the following error [[ref](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=459102&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=1366)]:
```bash
x86_64-conda-linux-gnu-cc: error: pysurvival/cpp_extensions/_functions.cpp: No such file or directory
```

Reply:
Thanks, I'll revert!